docs/configuration/adding_your_own.md

```diff
@@ -0,0 +1,86 @@
+# nf-core/neutronstar: Configuration for other clusters
+
+It is entirely possible to run this pipeline on other clusters, though you will need to set up your own config file so that the pipeline knows how to work with your cluster.
```

Comment:
This is on the main website note and can be removed.

Reply:
It's from the template. But it's gone now :dash:

Reply:
Or is this used for the `output_documentation` process?